src/adapters/simple_interest_loan_adapter.ts

```diff
@@ -356,13 +356,15 @@ export class SimpleInterestLoanAdapter {
             principalTokenIndex,
         );
 
-        return {
+        const loanOrder: SimpleInterestLoanOrder = {
```

Comment:
don't really need the local variable but up to you. This definitely improves readability.